src/roktManager.ts

```diff
@@ -97,6 +111,22 @@ export default class RoktManager {
         return Boolean(this.kit && this.kit.launcher);
     }
 
+    private mapUserAttributes(attributes: IRoktPartnerAttributes, userAttributeMapping: Dictionary<string>[]): IRoktPartnerAttributes {
```

Comment:
So the logic seems fine I just had a harder time understanding it because IRoktPartnerAttributes. It's just an interface of a dictionary right? Does it do something useful right now or it more for supporting future changes?

Reply:
It just makes sure we conform to the expected interface from Rokt: https://docs.rokt.com/developers/integration-guides/web/library/attributes/